test/e2e/allocator_test.go

```diff
@@ -61,8 +61,17 @@ func TestAllocator(t *testing.T) {
 	tlsCA := refreshAllocatorTLSCerts(t, ip)
 
 	namespace := fmt.Sprintf("allocator-%s", uuid.NewUUID())
-	framework.CreateNamespace(t, namespace)
-	defer framework.DeleteNamespace(t, namespace)
+
```

Comment:
Instead of creating a new namespace, it looks like we can reuse `namespace := framework.Namespace`